integration/ec2_test.go

```diff
@@ -50,6 +55,26 @@ func newNodeConfig(t *testing.T, authAddr utils.NetAddr, awsTokenName string) *s
 	return config
 }
 
+func newProxyConfig(t *testing.T, authAddr utils.NetAddr, tokenName string, joinMethod types.JoinMethod) *service.Config {
+	config := service.MakeDefaultConfig()
```

Comment:
Could we silence logging for the tests? Ditto for the one above.

I think it needs a combination of [configuring the logger](https://github.com/gravitational/teleport/blob/d012a0b6d96cffd05e368dabab3febe94bf020b2/lib/client/api_login_test.go#L234) and [setting it in the config](https://github.com/gravitational/teleport/blob/d012a0b6d96cffd05e368dabab3febe94bf020b2/lib/client/api_login_test.go#L256). (It's been a while so I might be missing something.)